sn_interface/src/network_knowledge/mod.rs

```diff
@@ -499,6 +489,11 @@ impl NetworkKnowledge {
         &self.prefix_map
     }
 
+    // Returns mutable reference to network prefix map
+    pub fn mut_prefix_map(&mut self) -> &mut NetworkPrefixMap {
```

Comment:
Convention is for `mut` on the end: `<name>_mut()`, i.e. `iter_mut()`

```suggestion
    pub fn prefix_map_mut(&mut self) -> &mut NetworkPrefixMap {
```